commands/stop.go

```diff
@@ -16,6 +26,37 @@ var StopCmd = &cli.Command{
 		}
 		defer closer()
 
+		if cctx.String("height") != "" {
```

Comment:
Is this necessary. Can't you always do `lily sync status` just before calling `lily stop` to get the same information?

Reply:
You're right, I'll remove this.